js/caption-loader.js

```diff
@@ -25,10 +25,15 @@ var captionsContainerId = "captions-container";
           }
       });
   }
-  
+
   function populateCaptions(game, div) {
       var captions = game.captions;
       var numDisplayed = 0;
+      if(captions === undefined){
+        captions = {};
+        captions[0] = {card : {cardText : "No top comment yet! Download Snaption to submit one!"},
+          userInput: {0 : ""}};
+      }
       for(var captionId in captions) {
```

Comment:
Should probably wrap this in an else, since it shouldn't be run if captions is undefined